### DISCUSSION TITLE
[TRILHA EXISTENTE] Github Start - Adição de um novo tópico

### DISCUSSION THREAD

**Discussion by LuizFernando-TC** [codaqui/institucional#76] [Trilhas de Ensino]
Acredito que seja bom adicionar uma explicação sobre o "Discussions". O que acham?

**Comment by endersonmenezes** [comment]
Não tinha pensado nisso, aliás nem o proprio GitHub eu acho que atualizou a trilha deles... O que adicionarmos na nossa, ainda podemos colocar aqui: https://github.com/education/github-starter-course

**Reply by endersonmenezes** [reply]
Olha o GIF que esse pessoal colocou aqui, um gif bem lentinho eu adorei o modelo de repositório deles: https://github.com/education/PR-Picafresas-Request

**Reply by endersonmenezes** [reply]
@thaifurforo resuma em uma frase o GitHub Discussions para você kkk

**Reply by endersonmenezes** [reply]
@LuizFernando-TC documentação técnica do GitHub Discussions: https://docs.github.com/pt/discussions

**Reply by LuizFernando-TC** [reply]
### Discussions (Discussão)

O GitHub Discussions é um fórum de comunicação colaborativa para a comunidade em torno de um projeto de código aberto ou interno. Os membros da comunidade podem fazer e responder perguntas, compartilhar atualizações, ter conversas abertas e acompanhar as decisões que afetam o modo de trabalho da comunidade. Para saber mais sobre discussões, leia ["Sobre Discussões"](https://docs.github.com/pt/discussions/collaborating-with-your-community-using-discussions/about-discussions).

O que acha, @endersonmenezes?
É o resumo do próprio GitHub, acho que está bom.
Obs: Coloquei [esse link](https://docs.github.com/pt/discussions/collaborating-with-your-community-using-discussions/about-discussions) para o "Sobre Discussões" para manter o padrão em relação aos outros tópicos, como o ["Sobre os Pull Requests"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)

**Reply by endersonmenezes** [reply]
Acho interessante, em que parte da trilha acha que ele se encaixa melhor? Depois que a pessoa já fez o processo e entendeu o GitFlow, ou antes como uma apresentação da ferramenta?

**Reply by LuizFernando-TC** [reply]
Acredito que depois do tópico de Pull Resquest fique bom, o que acha?

**Reply by LuizFernando-TC** [reply]
Acredito que dá para colocar mais esse texto:
Com as discussões do GitHub, a comunidade do seu projeto pode criar e participar de conversas no repositório ou organização do projeto. As discussões capacitam os mantenedores, colaboradores e visitantes de um projeto a reunir e atingir metas em um local central, sem ferramentas de terceiros.
Você pode usar as discussões do repositório para discutir tópicos específicos do repositório. Se o seu projeto abrange vários repositórios, você pode usar as discussões da organização para discutir tópicos que não são específicos de um único repositório em sua organização.
Alguns desses...
